root/scripts/talker/coach.txt

```diff
@@ -4941,6 +4964,7 @@ Rule SurvivorSpottedWorldCloseCoach
 Response SurvivorSpottedArmoredCoach
 {
 	norepeat
```

Comment:
Already fixed to use generic rules to be used on specific maps when the first one is used. Pretty sure I've uploaded that.

Reply:
I liked how c1m1 and c5m1 had unique callouts for uncommon infected -- it reduced the repetition of these voice lines. The problem with the TLS uncommon-infected callouts was that the survivor team couldn't say more than one callout per map -- which mostly affected only the Clowns and Mudmen.

Reply:
Fixed in new commit.